deployment/kubernetes/ktranslate-nr.yaml

```diff
@@ -28,9 +31,11 @@ spec:
               key: nr_account_id
         args:
           - --metalisten=0.0.0.0:8083
-          - --snmp=/etc/ktranslate/snmp.yml
+          - --snmp=/etc/ktranslate/snmp-base.yaml
           - --metrics=jchf
           - --tee_logs=true
+          - --snmp_discovery_on_start=true
+          - --snmp_discovery_min=360
```

Comment:
I was (somewhat arbitrarily) thinking 1 hour, but 6 is good. We can adapt our UI to do the same